pkg/auth/service.go

```diff
@@ -699,8 +689,8 @@ func (s *DBAuthService) ListPolicies(ctx context.Context, params *model.Paginati
 }
 
 func (s *DBAuthService) CreateCredentials(ctx context.Context, username string) (*model.Credential, error) {
-	accessKeyID := genAccessKeyID()
-	secretAccessKey := genSecretAccessKey()
+	accessKeyID := compat.GenAccessKeyID()
+	secretAccessKey := compat.GenSecretAccessKey()
```

Comment:
```suggestion
	accessKeyID := keys.GenAccessKeyID()
	secretAccessKey := keys.GenSecretAccessKey()
```
?

Reply:
Yikes!  Bad rename, sorry (and thanks...)